.github/workflows/pypi-publish.yml

```diff
@@ -0,0 +1,32 @@
+# .github/workflows/pypi-publish.yml
+name: Publish to PyPI
+
+on:
+  release:
+    types: [ created ]
+
+jobs:
+  deploy:
+    if: github.actor == 'nficano' || github.actor == 'RONNCC' || github.actor == 'tfdahlin'
+    name: Build Python distribution and publish to PyPI
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repo
+        uses: actions/checkout@v2
+      - name: Setup python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install build setuptools wheel twine
+
+      - name: Build and publish
+        env:
+          TWINE_USERNAME: __token__
```

Comment:
does it bump version somewhere ...?

Reply:
@RONNCC this workflow will only trigger when a GitHub release is made, which can be done from [this page](https://github.com/pytube/pytube/releases) by admins. Our workflow for this would be to merge a PR, use `bump2version` to handle version bumping, then come to the repo and create the release. 

I'm hesitant to use an automatic release creation because I saw it getting triggered by dependabot a lot in... weird ways. Since pytube doesn't have any direct dependencies (the pipfile only has dev-packages, and its all linting stuff), any time a dependency it didn't rely on had a minor/major version update, it would create minor/major release for pytube which doesn't make sense since nothing actually changed for our users.

Reply:
I don't think it makes sense to have automated version bumping unless we get it right the very first time, because I think it would be easy to accidentally make major version releases when we really shouldn't

Reply:
ah i see ok